server/handlers/events_streamer.go

```diff
@@ -22,6 +22,13 @@ var (
 	flusherMap map[string]http.Flusher
 )
 
+// swagger:route GET /api/events/events-streamer EventsAPI idGetEventStreamer
+// Handle GET request for events streamer
+//
+// Fetches events from Remote Provider and streams it to the UI
+// responses:
+// 	200:
+
```

Comment:
No parameters listed and no response object definition?

Reply:
Most of the others are like this.

This is why I mentioned that if this is going to be the case here, this type of change needs to be reflected across the rest of the endpoints.